teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -131,194 +134,163 @@ surfacing the verification information and building a corpus of verified resourc
-Tekton Hub can indicate the source Catalog of each resource, such as
-through a badge or tag. 
-
-To add the Official Catalog, we only have to modify the Catalogs 
-[configuration][hub-config] in the Hub, as such:
+* _Bandwidth_: The [Tekton Catalog Maintainers][catalog-owners] approve its addition based on its usefulness to the
+community and existing bandwidth to maintain the resource.
 
-```yaml
-catalogs:
-  
-  # community catalog
-  - name: tekton-community
-    org: tektoncd
-    type: community
-    provider: github
-    url: https://github.com/tektoncd/catalog
-    revision: main
-    
-  # official catalog  
-  - name: tekton-official
-    org: tektoncd
-    type: official
-    provider: github
-    url: https://github.com/tektoncd/catalog-official
-    revision: main
-```
+#### Hub
 
-This provides extensibility to the support tiers: we can add additional
-Catalogs for other support tiers as needed in the future. 
+Users rely on the [Tekton Hub][hub] to discover shared resources. The Tekton Hub supports publishing resources from
+multiple Catalogs. Users and organizations can create their own Catalogs and share them in the Tekton Hub, as long as
+they comply with the Catalog contract. The Tekton Hub can indicate the source Catalog of each resource, such as through
+a badge or tag. 
 
-#### Community and Official Catalogs in the Tekton CLI
+To add Catalogs, we only have to modify the Catalogs [configuration][hub-config] in the Hub, as such:
 
-Today, users can use the CLI to install resources from Catalogs by
-passing in the Catalog name to the `--from` argument as shown in the
-examples below. This will add a label to the resource indicating the
-source Catalog: `hub.tekton.dev/catalog=<catalog-name>`.
+```yaml
+catalogs:
+
+# community catalog (existing one)
+- name: tekton
+  org: tektoncd
+  type: community
+  provider: github
+  url: https://github.com/tektoncd/catalog
+
+# community catalog from buildpacks
+- name: buildpacks
+  org: buildpacks
+  type: community
+  provider: github
+  url: https://github.com/buildpacks/tekton-integration
+  versioning: git
+
+# community catalog from eBay
+- name: slack-notify
+  org: ebay
+  type: community
+  provider: github
+  url: https://github.com/eBay/tekton-slack-notify
+  versioning: git
+
+# community catalog from Openshift
+- name: openshift
+  org: redhat
+  type: community
+  provider: github
+  url: https://github.com/openshift/tekton-catalog
+  versioning: git
+
+# official catalog for git resources
+- name: git
+  org: tektoncd-catalog
+  type: official
+  provider: github
+  url: https://github.com/tektoncd-catalog/git
+
+# official catalog for kaniko resources
+- name: kaniko
+  org: tektoncd-catalog
+  type: official
+  provider: github
+  url: https://github.com/tektoncd-catalog/kaniko
+ ```
+
+We could remove the existing `type` field because it should only be set to `"official"` when the Catalog is from the 
```

Comment:
is there a way to actually enforce the catalogs with type = official actually meet the requirements stated above?